Example/StripeConnectExample/StripeConnectExample/Appearance/AppSettings+AppearanceInfo.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AppSettings+AppearanceInfo.swift
+//  StripeConnect Example
+//
+//  Created by Chris Mays on 9/5/24.
+//
+
+import Foundation
+@_spi(PrivateBetaConnect) import StripeConnect
+import UIKit
+
+extension AppSettings {
+    var appearanceOptions: [AppearanceInfo] {
+        [
+            .default,
+            .hotDog,
+            .link,
+            .oceanBreeze,
+            .shrek
+        ]
+    }
+    
+    var appearanceInfo: AppearanceInfo {
+        get {
+            appearanceOptions.first(where: {
+                $0.id == AppSettings.shared.appearanceId
+            }) ?? .default
+        }
+        set {
+            appearanceId = newValue.id
+        }
+    }
+}
+
+
+extension AppearanceInfo {
+    static var `default`: AppearanceInfo {
+        .init(displayName: "Default", appearance: .default)
+    }
+    
+    static var shrek: AppearanceInfo {
+        .init(displayName: "Shrek", appearance: .fromURL("https://stripe.com/docs/connect/customize-connect-embedded-components?enable_embedded_theming_demo=true&enable_uiconfig_copy_link=true&enable_new_theming=true&initOptionsOverride=%7B%22appearance%22%3A%7B%22colorPrimary%22%3A%22%235AE92B%22%2C%22colorBackground%22%3A%22%23837411%22%2C%22colorPrimaryButtonBackground%22%3A%22%23DAB9B9%22%2C%22colorPrimaryButtonBorder%22%3A%22%23F00000%22%2C%22colorPrimaryButtonText%22%3A%22%23000000%22%2C%22colorSecondaryButtonBackground%22%3A%22%23025F08%22%2C%22colorText%22%3A%22%23554125%22%2C%22badgeNeutralTextColor%22%3A%22%2328D72A%22%2C%22badgeNeutralBackgroundColor%22%3A%22%23638863%22%2C%22colorSecondaryButtonText%22%3A%22%23000000%22%2C%22fontFamily%22%3A%22fantasy%22%7D%7D")!)
```

Comment:
I'm hesitant hardcode the URL encoding from our docs site as a means of encoding the theme. The URL compatibility is not something we guarantee won't change in the future and we want to encourage best practices here as this is an example app for platform integrators to see how to use the SDK.

If the intent is to demonstrate how to easily translate the appearance from the appearance playground, then what I recommend doing instead is:
1. Hardcode the appearance using the `Appearance` object rather than decoding the URL
2. Add a comment above like "[You can view this appearance in our playground]({url})"
3. We can update the playground to also generate Swift code instead of just JS appearance code (see the bottom section in the screenshot below). We'll want to do this when we eventually publicly GA the SDK anyway.

<img width="1606" alt="image" src="https://github.com/user-attachments/assets/e1cc6415-6e73-49b1-850e-9ad88227c9be">